### ISSUE TITLE
Fix split scenario with 1.0.8 release

### ISSUE THREAD

**Issue by aszlig** [thrillfall/nextcloud-gpodder#14]
Version 1.0.8 doesn't have a real release in the [app store](https://apps.nextcloud.com/apps/gpoddersync/releases) (it still says 1.0.7) but the upstream tarball URLs have changed to `https://github.com/thrillfall/nextcloud-gpodder/raw/main/releases/gpoddersync-1.0.8.tar.gz`.

This means that for new Nextcloud instances you get version 1.0.8 but old Nextcloud instances will not get updated and stay at version 1.0.7.

**Comment by thrillfall** [comment]
How can that happen? I uploaded the 1.0.8. release via web upload and there was no error.
Its like I updated only the link for the 1.0.7 release

**Comment by aszlig** [comment]
If you download <https://github.com/thrillfall/nextcloud-gpodder/raw/main/releases/gpoddersync-1.0.8.tar.gz> (as linked on the app store) and look into `appinfo/info.xml`, you'll see:

```xml
<?xml version="1.0"?>
<info xmlns:xsi= "http://www.w3.org/2001/XMLSchema-instance"
      xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/info.xsd">
    ...
    <version>1.0.7</version>
    ...
</info>
```

I guess the tarball probably got created before bumping the version.

<details>
<summary>Something off topic...</summary>

That aside, it's your project and you can use any workflow you like, but Git is not very good when it comes to handling binary files. There are solutions such as [git-annex](https://git-annex.branchable.com/) or [Git LFS](https://docs.github.com/en/github/managing-large-files/versioning-large-files/about-git-large-file-storage), but especially when it comes to release tarballs, GitHub already provides these if you properly tag your releases so adding them to the repository is not only redundant but also unnecessarily increases repository size.

For example to get the tarball for tag `0.0.2` you can use <https://github.com/thrillfall/nextcloud-gpodder/archive/refs/tags/0.0.2.tar.gz>. Of course, the obvious downside is that it doesn't fare very well with app signatures, but for that you can upload custom release tarballs, eg. if you look [here](https://github.com/thrillfall/nextcloud-gpodder/releases) you should have a way to edit your releases and attach files.

And well, of course it would also make sense to automate that whole process in the long term, but I don't want to go off topic for too long now.
</details>

**Comment by thrillfall** [comment]
Thanks a lot for finding the issue here (being my manual workflow :-) )

**Comment by thrillfall** [comment]
TL;DR: how can I (manually) add a tarball to a release?

I tried using the release tarballs from github but they weren't accepted because of wrong directory name (app name differs from github project name).

I also did not find an easy way to add/upload custom files to a release. Only some automated github workflow CD stuff that clearly exceeded the scope of this app that I deemed ready and bug free (ahahaha).

That's why I resorted to versioning the tarballs.
